woocommerce/woocommerce-gutenberg-products-block
# Migrate storybook entries of ProductPrice component

> ⚠️ This PR is based on #5324 and that's why the base is set as that. Once that is approved, this branch will be rebased on `trunk` and merged.

This PR migrates stories for the `ProductPrice` component and adds documentation for its props.

Fixes #5368

### Testing

### Manual Testing

How to test the changes in this Pull Request:

1. Run `npm run storybook`
2. Explore the [ProductPrice stories](http://localhost:6006/?path=/story/woocommerce-blocks-base-components-productprice--default). There should be three stories: one for the regular component, one configured for the price range and one for the price on sale.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
